### ISSUE TITLE
[EDIT] automatic tracks synching by audio aligning

### ISSUE THREAD

**Issue by forart** [olive-editor/olive#1649]
It would be really cool - and almost uniqe, at least between open source NLEs - to have an integrated function that helps editors to automatically synch multitracks in a snap.

I found some interesting resources that may help to integrate it:
- https://github.com/protyposis/AudioAlign - a tool written for research purposes to automatically synchronize audio and video recordings that have either been recorded in parallel at the same event or contain the same aural information.
- https://github.com/alopatindev/sync-audio-tracks - a tool (based on [Miles McCoo's cross-correlation implementation](https://web.archive.org/web/20170916044116/https://nerd.mmccoo.com/2017/06/19/automatically-aligning-multiple-videoaudio-clips-in-kdenlive/)) that calculates a delay between two audios and produces a shifted audio.
- https://github.com/benfmiller/audalign - Python package for aligning audio files using audio fingerprinting.
- https://github.com/recastrodiaz/align-audio-recordings-julia - Aligning video recordings with Julia

Hope that inspires.

**Comment by Simran-B** [comment]
"Acoustic" / audio signal-based track alignment is certainly handy, but not a priority. Timecode-based alignment is an order of magnitude simpler I guess and more important for a somewhat serious production. I'm not sure if it's still a 0.2 target, however.

https://github.com/olive-editor/olive/projects/

> Longitudinal Time Code is an important library to integrate as audiences familiar with footage will almost exclusively use an external recorder for audio. LibLTC is the likely target for use here, permitting synchronization of external recorder audio. Also requires metadata analysis for timecode parcels.

https://github.com/olive-editor/olive/projects/7#column-8981234

https://github.com/x42/libltc

**Comment by itsmattkc** [comment]
Duplicate of #365
